### PR TITLE
Update two informtion_schema tables desc

### PR DESCRIPTION
**Pull request by yibin87**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update processlist and slow_query tables' desc, both add two new columns.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions))

### What is the related PR or file link(s)?
https://github.com/pingcap/tidb/pull/55455

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-processlist.md [hunk 1]
```diff
@@ -85,6 +91,8 @@ RESOURCE_GROUP: default
 * `RESOURCE_GROUP` 列：显示对应的资源组名称。
 * `SESSION_ALIAS` 列：显示当前连接的别名。
 * `ROWS_AFFECTED` 列：显示语句当前影响的行数。
+* `TIDB_CPU` 列：显示语句当前占用 `TIDB` 服务器 CPU 时间，单位是秒。
+* `TIKV_CPU` 列：显示语句当前占用 `TIKV` 服务器 CPU 时间，单位是秒。
```
Comment (hfxsd) [inline information-schema/information-schema-processlist.md:95]:
```suggestion
* `TIDB_CPU` 列：显示语句当前占用 TiDB 服务器 CPU 时间，单位是秒。
* `TIKV_CPU` 列：显示语句当前占用 TiKV 服务器 CPU 时间，单位是秒。
```

File: information-schema/information-schema-processlist.md [hunk 2]
```diff
@@ -85,6 +91,8 @@ RESOURCE_GROUP: default
 * `RESOURCE_GROUP` 列：显示对应的资源组名称。
 * `SESSION_ALIAS` 列：显示当前连接的别名。
 * `ROWS_AFFECTED` 列：显示语句当前影响的行数。
+* `TIDB_CPU` 列：显示语句当前占用 TiDB 服务器 CPU 时间，单位是秒。
```
Comment (xzhangxian1008) [inline information-schema/information-schema-processlist.md:94]: Maybe we can take an example to explain what does the cpu time means and how we calculate the cpu time.

File: information-schema/information-schema-processlist.md [hunk 3]
```diff
@@ -97,9 +105,9 @@ SELECT * FROM information_schema.cluster_processlist;
...
+| INSTANCE        | ID         | USER | HOST            | DB   | COMMAND | TIME | STATE      | INFO                                                 | DIGEST                                                           | MEM  | DISK | TxnStart                               | RESOURCE_GROUP | SESSION_ALIAS | ROWS_AFFECTED | TIDB_CPU | TIKV_CPU |
++-----------------+------------+------+-----------------+------+---------+------+------------+------------------------------------------------------+------------------------------------------------------------------+------+------+----------------------------------------+----------------+---------------+---------------+----------+----------+
+| 127.0.0.1:10080 | 1268776964 | root | 127.0.0.1:59922 | NULL | Query   |    0 | autocommit | SELECT * FROM information_schema.cluster_processlist | b1e38e59fbbc3e2b35546db5c8053040db989a497ac6cd71ff8dd4394395701a |    0 |    0 | 07-29 12:39:24.282(451471727468740609) | default        |               |             0 |        0 |        0 |
```
Comment (xzhangxian1008) [inline information-schema/information-schema-processlist.md:111]: Maybe we can show an example that `TIDB_CPU` and `TIKV_CPU` are not zero.

Comment (yibin87) [reply information-schema/information-schema-processlist.md:111]: Several columns of the previous example are also 0/null values, and it makes sense to be 0,0 for "select * from information_schema.cluster_processlist", so prefer to keep it zero.

File: information-schema/information-schema-processlist.md (hunk 2 above)
Comment (yibin87) [reply information-schema/information-schema-processlist.md:94]: Explain it depends on topSQL feature to collect tidb cpu now.

**Review by xzhangxian1008 — APPROVED** [review]
LGTM

File: information-schema/information-schema-processlist.md [hunk 4]
```diff
@@ -17,6 +17,8 @@ aliases: ['/docs-cn/dev/information-schema/information-schema-processlist/']
 * `RESOURCE_GROUP` 列：显示对应的资源组名称。
 * `SESSION_ALIAS` 列：显示当前连接的别名。
 * `ROWS_AFFECTED` 列：显示语句当前影响的行数。
+* `TIDB_CPU` 列：显示语句当前占用 TiDB 服务器 CPU 时间，单位是秒。例如某查询持续占满 tidb 服务器的4个 CPU 1秒钟，那么该值为4。该列仅在开启 [Top SQL 特性](/dashboard/top-sql.md) 时有非0值。
```
Comment (songrijie) [inline information-schema/information-schema-processlist.md]: "秒" 作为单位会不会太粗了？以我们能计量的最小时间单位比如 纳秒，用整数来显示会不会更好？

File: information-schema/information-schema-processlist.md (hunk 3 above)
Comment (songrijie) [reply information-schema/information-schema-processlist.md:111]: I agreed with @xzhangxian1008 that we'd better have a sample outputs here.

File: information-schema/information-schema-slow-query.md [hunk 5]
```diff
@@ -92,6 +92,8 @@ DESC slow_query;
 | Request_unit_read             | double              | YES  |      | NULL    |       |
 | Request_unit_write            | double              | YES  |      | NULL    |       |
 | Time_queued_by_rc             | double              | YES  |      | NULL    |       |
+| Tidb_cpu_usage                | double              | YES  |      | NULL    |       |
+| Tikv_cpu_usage                | double              | YES  |      | NULL    |       |
```
Comment (songrijie) [inline information-schema/information-schema-slow-query.md:96]: I prefer "TiDB_cpu_time" and "Tikv_cpu_time" in slow logs.

File: information-schema/information-schema-processlist.md (hunk 4 above)
Comment (yibin87) [reply information-schema/information-schema-processlist.md]: 嗯 对长一点的查询的话，秒可能读起来舒服点，短一点的话，纳秒好一点。确实纳秒会更精确点

Comment (songrijie) [reply information-schema/information-schema-processlist.md]: 做显示和聚合的时候，也会更容易一些。 因为这个数值本身其实意义不大，主要是排序比大小。

File: information-schema/information-schema-processlist.md [hunk 6]
```diff
@@ -85,6 +91,8 @@ RESOURCE_GROUP: default
 * `RESOURCE_GROUP` 列：显示对应的资源组名称。
 * `SESSION_ALIAS` 列：显示当前连接的别名。
 * `ROWS_AFFECTED` 列：显示语句当前影响的行数。
+* `TIDB_CPU` 列：显示语句当前占用 TiDB 服务器 CPU 时间，单位是纳秒。
```
Comment (songrijie) [inline information-schema/information-schema-processlist.md:94]:
```suggestion
* `TIDB_CPU` 列：显示语句当前占用 TiDB 服务器 CPU 时间，单位是纳秒。该列仅在开启 [Top SQL 特性](/dashboard/top-sql.md)时显示实际值，否则始终显示为 `0`。
```

**Review by songrijie — APPROVED** [review]
rest LGTM

File: information-schema/information-schema-processlist.md [hunk 7]
```diff
@@ -17,6 +17,8 @@ aliases: ['/docs-cn/dev/information-schema/information-schema-processlist/']
 * `RESOURCE_GROUP` 列：显示对应的资源组名称。
 * `SESSION_ALIAS` 列：显示当前连接的别名。
 * `ROWS_AFFECTED` 列：显示语句当前影响的行数。
+* `TIDB_CPU` 列：显示语句当前占用 TiDB 服务器 CPU 时间，单位是纳秒。该列仅在开启 [Top SQL 特性](/dashboard/top-sql.md)时显示实际值，否则始终显示为 `0`。
```
Comment (lilin90) [inline information-schema/information-schema-processlist.md:20]:
```suggestion
* `TIDB_CPU` 列：显示语句当前占用 TiDB 服务器 CPU 的时间，单位是纳秒。该列仅在开启 [Top SQL 特性](/dashboard/top-sql.md)时显示实际值，否则始终显示为 `0`。
```

File: information-schema/information-schema-processlist.md [hunk 8]
```diff
@@ -17,6 +17,8 @@ aliases: ['/docs-cn/dev/information-schema/information-schema-processlist/']
...
+* `TIKV_CPU` 列：显示语句当前占用 TiKV 服务器 CPU 时间，单位是纳秒。
```
Comment (lilin90) [inline information-schema/information-schema-processlist.md:21]:
```suggestion
* `TIKV_CPU` 列：显示语句当前占用 TiKV 服务器 CPU 的时间，单位是纳秒。
```

File: information-schema/information-schema-processlist.md [hunk 9]
```diff
@@ -85,6 +91,8 @@ RESOURCE_GROUP: default
 * `RESOURCE_GROUP` 列：显示对应的资源组名称。
 * `SESSION_ALIAS` 列：显示当前连接的别名。
 * `ROWS_AFFECTED` 列：显示语句当前影响的行数。
+* `TIDB_CPU` 列：显示语句当前占用 TiDB 服务器 CPU 时间，单位是纳秒。该列仅在开启 [Top SQL 特性](/dashboard/top-sql.md)时显示实际值，否则始终显示为 `0`。
```
Comment (lilin90) [inline information-schema/information-schema-processlist.md:94]:
```suggestion
* `TIDB_CPU` 列：显示语句当前占用 TiDB 服务器 CPU 的时间，单位是纳秒。该列仅在开启 [Top SQL 特性](/dashboard/top-sql.md)时显示实际值，否则始终显示为 `0`。
```

File: information-schema/information-schema-processlist.md [hunk 10]
```diff
@@ -85,6 +91,8 @@ RESOURCE_GROUP: default
...
+* `TIKV_CPU` 列：显示语句当前占用 TiKV 服务器 CPU 时间，单位是纳秒。
```
Comment (lilin90) [inline information-schema/information-schema-processlist.md:95]:
```suggestion
* `TIKV_CPU` 列：显示语句当前占用 TiKV 服务器 CPU 的时间，单位是纳秒。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**